### PR TITLE
Adds MDS code, configurable links, and uses gzip to compress the similarity matrix.

### PR DESCRIPTION
**Pull request by jherwig1**

### CODE REVIEW

**Comment by jduggan1** [comment]
What's the compression rate using gzip?

**Comment by jherwig1** [comment]
The file goes from ~8 megabytes to ~2 megabytes

File: denvercode/flexview/assets/js/mds_util.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+define(['jquery', 'd3', 'spin', 'mds'], function($, d3, Spinner, mds) {
+return {
+	state : {
+		"view": null
+	},
+
+	run_mds: function() {
+		var self = this;
+		var spinner = new Spinner({color: "#444", lines: 20, scale: 2}).spin(
+			document.getElementsByClassName("mds")[0]
```
Comment (jduggan1) [inline denvercode/flexview/assets/js/mds_util.js]: Probably easier to just use `$(".mds").get(0);`, but this is fine.

File: denvercode/flexview/npmap/flexview/views.py [hunk 2]
```diff
@@ -1,6 +1,10 @@
+from django.http import HttpResponse, Http404
 from django.conf import settings
 from django.shortcuts import render
-from django.views.decorators.http import require_GET
+from django.views.decorators.http import require_GET, require_POST
+from django.views.decorators.gzip import gzip_page
```
Comment (jduggan1) [inline denvercode/flexview/npmap/flexview/views.py]: This is a cool feature of Django.

**Comment by jduggan1** [comment]
One more question, do you have any way to time how long the decompression takes? It's not a huge deal, just curious if you have a rough idea.

**Comment by jherwig1** [comment]
I'm not sure how long the decompression takes, but the overall time for loading the file decreases.

**Pull request merged**